node/src/main/scala/com/wavesplatform/state/diffs/invoke/InvokeScriptDiff.scala

```diff
@@ -57,6 +57,13 @@ object InvokeScriptDiff {
     val result = blockchain.accountScript(dAppAddress) match {
       case Some(AccountScriptInfo(pk, ContractScriptImpl(version, contract), _, callableComplexities)) =>
         for {
+          _ <- traced(
+            Either.cond(
+              version >= V5,
+              (),
+              GenericError(s"Calling DApp is available only from V5, but DApp $version at address $dAppAddress was called from $invoker")
```

Comment:
I suggest to replace error message with this: "DApp $invoker invoked DApp $dAppAddress that uses Ride version $version, but dApp-to-dApp invocation requires version 5 or higher."

Reply:
Done